Lombiq.Tests.UI/Extensions/VisualVerificationUITestContextExtensions.cs

```diff
@@ -166,6 +300,12 @@ public static class VisualVerificationUITestContextExtensions
         var configuration = new VisualVerificationMatchApprovedConfiguration();
         configurator?.Invoke(configuration);
 
+        if (configuration.Platforms?.Any(platform => platform == Environment.OSVersion.Platform) is false)
+        {
+            context.Configuration.TestOutputHelper.WriteLineTimestampedAndDebug("Test skipped based on configuration.");
```

Comment:
Keep in mind that this text will be printed in the middle of the test output without much further clue. Give some context.